### PR TITLE
add `notify: true` to marker, latitude, and longitude properties

### PR DESCRIPTION
**Pull request by naderio**
added `notify: true`:
* to `latitude` and `longitude` properties to solve draggable marker issue (#119, #207)
* to `marker` property to enable direct marker manipulation

### CODE REVIEW

File: google-map-marker.html [hunk 1]
```diff
@@ -192,7 +195,8 @@
       longitude: {
         type: Number,
         value: null,
-        reflectToAttribute: true
+        reflectToAttribute: true,
```
Comment (ebidel) [inline google-map-marker.html]: what's the use case for these to reflect? You only need to reflect when you want to style something with an attribute selector: `google-map-marker[longitude="123"]`

Comment (naderio) [inline google-map-marker.html]: agree, I don't see the point of attribute reflection for those properties.

Comment (naderio) [inline google-map-marker.html]: @ebidel should I remove the `reflectToAttribute: true` for latitude and longitude properties?

Comment (ebidel) [inline google-map-marker.html]: Yea, they're not needed.

**Comment by naderio** [comment]
@ebidel PR updated, removed `reflectToAttribute: true`

**Comment by ebidel** [comment]
LGTM

**Comment by matdehaast** [comment]
When will this be pushed to the polymer elements?

**Comment by ebidel** [comment]
I'll ask for a catalog update

**Pull request merged**